src/main/java/io/confluent/idesidecar/restapi/models/graph/CCloudFlinkComputePool.java

```diff
@@ -0,0 +1,30 @@
+package io.confluent.idesidecar.restapi.models.graph;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import org.eclipse.microprofile.graphql.Name;
+
+@RegisterForReflection
+public record CCloudFlinkComputePool(
+    @JsonProperty("id") String id,
+    @JsonProperty("display_name") String displayName,
+    @JsonProperty("provider") @Name("provider") String provider,
```

Comment:
We probably don't need the annotation anymore.

```suggestion
    @JsonProperty("provider") String provider,
```